includes/admin/class-sensei-setup-wizard-pages.php

```diff
@@ -67,6 +64,10 @@ public function create_pages() {
 		$new_my_course_page_id = $this->create_page( esc_sql( _x( 'my-courses', 'page_slug', 'sensei-lms' ) ), __( 'My Courses', 'sensei-lms' ), $this->get_learner_courses_page_content() );
 		Sensei()->settings->set( 'my_course_page', $new_my_course_page_id );
 
+		// Course Completion Page.
+		$new_course_completion_page_id = $this->create_page( esc_sql( _x( 'course-completion', 'page_slug', 'sensei-lms' ) ), __( 'Course Completion', 'sensei-lms' ) );
```

Comment:
I wonder if the default title should be `Course Completed`?

Reply:
Yeah, that's better. I also updated all references in https://github.com/Automattic/sensei/pull/4254/commits/739a0c97412fc45bd7be2ec438ae466df093b779.